src/handlers/__tests__/deeplinks.test.ts

```diff
@@ -19,11 +19,7 @@ import {
 import { fetchReverseRecordWithRetry } from '@/utils/profileUtils';
 
 jest.mock('@/redux/store');
-jest.mock('@/redux/walletconnect', () => ({
-  walletConnectOnSessionRequest: jest.fn(),
-  walletConnectRemovePendingRedirect: jest.fn(),
-  walletConnectSetPendingRedirect: jest.fn(),
-}));
+jest.mock('@/redux/walletconnect');
```

Comment:
Moved mocks adjacent to these files, so no need to write the mock implementations here anymore.